File: src/main/java/no/nav/permitteringsskjemaapi/permittering/OpprettPermitteringsskjema.java
package no.nav.permitteringsskjemaapi.permittering;

import lombok.Value;

import java.util.List;

@Value
public class OpprettPermitteringsskjema {
    String bedriftNr;
    PermitteringsskjemaType type;
    List<String> underenheter;
}

File: src/main/kotlin/no/nav/permitteringsskjemaapi/permittering/deprecated/OpprettPermitteringsskjema.kt
package no.nav.permitteringsskjemaapi.permittering.deprecated

import no.nav.permitteringsskjemaapi.permittering.PermitteringsskjemaType


data class OpprettPermitteringsskjema(
    var bedriftNr: String? = null,
    var type: PermitteringsskjemaType? = null,
)